misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -596,19 +630,30 @@ public ImportPrefixNode transform(ImportPrefixNode importPrefixNode) {
     public ServiceDeclarationNode transform(ServiceDeclarationNode serviceDeclarationNode) {
         MetadataNode metadata = formatNode(serviceDeclarationNode.metadata().orElse(null), 0, 1);
         Token serviceKeyword = formatToken(serviceDeclarationNode.serviceKeyword(), 1, 0);
```

Comment:
The qualifiers [1] need to be formatted here as well right?

[1] https://github.com/ballerina-platform/ballerina-lang/blob/f919b1d55a84b1d302f188e657cf5155755c0327/compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/tree/TreeModifier.java#L157

Reply:
Fixed.